### DISCUSSION TITLE
@tailwindcss/forms default styles customization

### DISCUSSION THREAD

**Discussion by Van4kk** [tailwindlabs/tailwindcss#12567] [Ideas]
Is there a way to customize plugin default styles. For example in my case I have to change the `focus:ring` color and shadow. I'm using the most not **"clear"**, writing it directly in `app.css` file. Is there a way to make these changes from `tailwind.config.js` file ??

Is there a method to customize default styles for `@tailwindcss/forms` plugin? In my specific situation, I need to modify the `focus:ring` color and shadow. Currently, I'm not entirely clear on the most effective approach, as I'm directly editing the `app.css` file. Is it possible to implement these changes from the `tailwind.config.js` file?

Thank you ! ❤️

**Comment by abdelhak002** [comment]
I believe that's the only solution, overriding the default plugin styles in the `css` file.
```css

  [type='text'],
  [type='email'],
  [type='url'],
  [type='password'],
  [type='number'],
  [type='date'],
  [type='datetime-local'],
  [type='month'],
  [type='search'],
  [type='tel'],
  [type='time'],
  [type='week'],
  [multiple],
  textarea,
  select {
    @apply focus:ring-indigo-400 focus:border-indigo-400 rounded-md;
  }
```
